kyuubi-ctl/src/main/scala/org/apache/kyuubi/ctl/cmd/log/LogBatchCommand.scala

```diff
@@ -46,41 +46,73 @@ class LogBatchCommand(cliConfig: CliConfig, restConfigMap: JMap[String, Object]
 
       var log: OperationLog = null
       var done = false
-      var batch: Batch = null
+      var batch: Batch = batchRestApi.getBatchById(batchId)
```

Comment:
could you transfer the batch report to `LogBatchCommand`.

It is for the corner case when jdbc metastore is down.

Reply:
https://github.com/apache/incubator-kyuubi/pull/2834

Metadata store might be unavailable if database is in maintenance window.

For that case, the batch session state is only accessible in memory.